Source/WebKit/Shared/Cocoa/XPCEndpoint.h

```diff
@@ -27,20 +27,21 @@
 
 #ifdef __cplusplus
 
-#include "WKDeclarationSpecifiers.h"
+#include <WebKit/WKBase.h>
 #include <wtf/OSObjectPtr.h>
+
 #include <wtf/spi/darwin/XPCSPI.h>
 
 namespace WebKit {
 
 class XPCEndpoint {
 public:
-    WK_EXPORT XPCEndpoint();
-    WK_EXPORT virtual ~XPCEndpoint() = default;
+    XPCEndpoint() = default;
```

Comment:
Why move the construction logic into `initialize()`? If it was to make tapi happy, maybe the above can help.

Reply:
This was just to reduce the number of exported symbols. With the tapi changes above, I think this change can be reverted.